### DISCUSSION TITLE
In the future, can I work with PHP?

### DISCUSSION THREAD

**Discussion by darkterminal** [erezsh/Preql#16] [Q&A]
This repository is excellent... it's really neat! Good work dude...

**Comment by erezsh** [comment]
Thanks! There are currently no plans to add support for PHP, but I'll consider it if there's enough demand.

Meanwhile, perhaps you could get by with connecting PHP to Preql with a rest interface, using the `serve_rest()` function: https://preql.readthedocs.io/en/latest/preql-modules.html#serve_rest

I intend to extend and improve the rest interface, so let me know if that would help.
